machine/v1beta1/types_azureprovider.go

```diff
@@ -437,11 +481,38 @@ type DiskEncryptionSetParameters struct {
 // SecurityProfile specifies the Security profile settings for a
 // virtual machine or virtual machine scale set.
 type SecurityProfile struct {
-	// This field indicates whether Host Encryption should be enabled
-	// or disabled for a virtual machine or virtual machine scale
-	// set. Default is disabled.
+	// This field indicates whether Host Encryption should be enabled or disabled for a virtual
```

Comment:
Since we are changing it anyway
```suggestion
	// encryptionAtHost indicates whether Host Encryption should be enabled or disabled for a virtual
```